cli/args/config_file.rs

```diff
@@ -524,6 +535,14 @@ impl ConfigFile {
     /// Filenames that Deno will recognize when discovering config.
     const CONFIG_FILE_NAMES: [&str; 2] = ["deno.json", "deno.jsonc"];
 
+    // todo(dsherret): in the future, we should enforce all callers
+    // to provide a resolved path
```

Comment:
What I've done in other projects is introduced a `ResolvedPath` struct, then force resolving that once and passing it around.

Reply:
That's a very good idea, we should do the same